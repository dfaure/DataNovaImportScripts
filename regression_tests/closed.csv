#Identifiant;Libellé_du_site;Date_calendrier;Plage_horaire_1;Plage_horaire_2;Plage_horaire_3;Plage_horaire_4;Plage_horaire_5;Plage_horaire_6;Plage_horaire_7;Heure_limite_dépôt_Courrier;Heure_limite_dépôt_Chrono;Heure_limite_dépôt_Colis;Libellé_de_la_fermeture;Bp_Plage_horaire_1;Bp_Plage_horaire_2;Bp_Plage_horaire_3;Bp_Plage_horaire_4;Bp_Plage_horaire_5;Bp_Plage_horaire_6;Bp_Plage_horaire_7;Bp_Libellé_de_la_fermeture
17690A;GROS CAP BP;2021-02-18;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-16;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-20;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-11;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-19;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-27;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-26;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-28;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-25;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-29;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-24;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-14;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-01;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-30;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-13;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-07;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-14;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-22;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-18;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-23;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-02;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-24;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-20;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-04;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-13;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-11;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-03;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-31;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-15;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-19;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-21;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-08;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-17;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-06;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-16;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-23;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-25;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-10;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-15;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-12;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-17;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-09;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-01-12;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-05;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-02;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-26;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-09;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-28;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-04;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-06;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-13;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-08;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-12;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-07;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-27;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-17;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-03;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-11;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-04-08;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-24;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-04-05;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-28;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-05;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-04-06;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-19;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-04-14;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-10;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-25;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-04-11;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-04-04;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-27;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-30;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-29;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-22;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-31;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-15;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-18;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-14;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-04-07;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-04-09;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-26;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-04-10;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-01;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-23;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-16;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-04-02;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-21;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-04-03;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-03-20;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-04-01;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-04-12;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-22;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-04-13;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-02-21;FERME;;;;;;;;;;;FERME;;;;;;;
17690A;GROS CAP BP;2021-04-15;FERME;;;;;;;;;;;FERME;;;;;;;
