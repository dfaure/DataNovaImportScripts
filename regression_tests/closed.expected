17690A|GROS CAP BP|closed
